awscli/alias.py

```diff
@@ -126,13 +131,17 @@ def name(self, value):
 
 
 class ServiceAliasCommand(BaseAliasCommand):
-    UNSUPPORTED_GLOBAL_PARAMETERS = [
-        'debug',
-        'profile'
-    ]
-
-    def __init__(self, alias_name, alias_value, session, command_table,
-                 parser, shadow_proxy_command=None):
+    UNSUPPORTED_GLOBAL_PARAMETERS = ('debug', 'profile')
```

Comment:
Is the type change here from `list` to `tuple` OK?

Reply:
Yeah, that should be fine. This should have been a tuple to begin with.